manifests/ingress.yaml.in

```diff
@@ -0,0 +1,44 @@
+#
+# This is two staged:
+# 1. Tell nginx where which potrs should be redirected to
+# 2. Open the relevant ports using a service
+
+# This configmap needs to know all ports which ngninx should redirect
+# In plain words: These are the nginx stream configurations
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tcp-services
+  namespace: kube-system
+data:
+  3128: "default/spice-proxy:3128"
+  8182: "default/virt-controller-service:8182"
+  8183: "default/virt-api-service:8183"
+  8184: "default/haproxy-service:8184"
+  8186: "default/virt-manifest-service:8186"
+---
+# This service needs to open all ports which should be publicly accessible
+# In plain words: This tells k8s to forward these ports to nginx
+apiVersion: v1
+kind: Service
+metadata:
+  name: ingress-nginx
+  namespace: kube-system
+spec:
+  externalIPs:
```

Comment:
Eventually. It would be great. I'd love it.
So far I think it's a bug in CNI which is preventing this (https://github.com/kubernetes-incubator/kube-aws/issues/704).

At least NodePort does not work, that's why I went with external IP for now.

I should add a FIXME somewhere so we get back to it …

Reply:
ugh... okay. we'll need a issue to track getting rid of ExternalIP for good.

Reply:
There you go, #542